dune

```diff
@@ -0,0 +1,3 @@
+(env
+ (_
+  (flags :standard -strict-sequence -safe-string -warn-error -a+8)))
```

Comment:
I would omit warn-error altogether, otherwise why this specific choice of warnings?

Reply:
This is to turn all warnings except for 8 (missing cases in pattern-matching) into non-fatal warnings to make development less obtrusive locally. I don't think it's important to keep this here. I'll remove.